analyzer-core/src/main/java/ffb/analyzer/core/pubsub/Publisher.java

```diff
@@ -0,0 +1,73 @@
+package ffb.analyzer.core.pubsub;
+
+import java.util.*;
```

Comment:
@david-hu23 we currently have AvoidStarImports set in the checkstyle. How do you feel about that?